### PR TITLE
Fix : Do refactoring in service , footer, header  section 

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Tasks**

- Header
- [x] when scroll down, the header should be scrollable, when scrolling up header will be appear
- [x] Add top space on the header. Should look like a current header
- Footer
- [x] I'll invest should be floatable
- [x] remove designrush icon
- [x] Make top space of I'll invest same as other inputs
- Service Section Mobile
- [x] Fix the image(number bullets) is looking blurred on the iOS device
- [x] Reduce the top and bottom space of the service section title on mobile devices
- Why Canopas
- [x] Fix the image(number bullets) is looking blurred on desktop and mobile devices
- [x] Check the quality of all the images, specifically on the latest iOS devices. If it's not as per needs, let's change it.
- [x] The clutch image in the client review is not clear.
- [x] keep button text size and button size common in a full page
- [x] Add cursor:pointer, wherever text is clickable like Read More

### CODE REVIEW

File: vue-frontend/src/components/partials/NewHeader.vue [hunk 1]
```diff
@@ -133,11 +135,12 @@ export default {
   },
   mounted() {
     window.addEventListener("scroll", this.handleScroll);
-    this.navContainerHeight = this.$refs.mainHeader.clientHeight;
+    this.navContainerHeight = this.$refs.mainHeader.clientHeight + 30;
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/partials/NewHeader.vue:138]: Space somewhat large, decrease it to 25 or 28

File: vue-frontend/src/components/contact/ContactFormFooter.vue [hunk 2]
```diff
@@ -107,28 +107,41 @@
...
+                <label
+                  class="tw-pb-2 2xl:tw-pb-8"
+                  :class="
+                    floatable
+                      ? 'tw-absolute tw-mt-[5rem] tw-pb-2 tw-mb-[7rem] tw-left-0 tw-z-[2] tw-text-white tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.375rem] md:tw-leading-[1.6875rem] lg:tw-text-[1.75rem] lg:tw-leading-[2.125rem] tw-transform tw--translate-y-4 tw-origin-[0] tw-scale-75 tw-duration-300 peer-focus:tw-text-white peer-placeholder-shown:tw-scale-100 peer-placeholder-shown:tw-translate-y-0 peer-focus:tw-scale-75 peer-focus:tw--translate-y-4'
+                      : '' || investSelected
+                      ? 'tw-absolute tw-bottom-10 tw-left-0 tw-z-[2] tw-text-white tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.375rem] md:tw-leading-[1.6875rem] lg:tw-text-[1.75rem] lg:tw-leading-[2.125rem] tw-transform tw--translate-y-4 tw-origin-[0] tw-scale-75 tw-duration-300 peer-focus:tw-text-white peer-placeholder-shown:tw-scale-100 peer-placeholder-shown:tw-translate-y-0 peer-focus:tw-scale-75 peer-focus:tw--translate-y-4'
+                      : ''
+                  "
+                  >I'll invest</label
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contact/ContactFormFooter.vue:139]: Screenshot from 2023-01-27 17-32-13
Can take ref from https://codepen.io/dannibla/pen/amgRNR
Fix top space. Also top space of `send me NDA ` is too much, make all fields top space identical
Refactor logic accordingly

File: vue-frontend/src/components/home-new/ServiceSectionMobile.vue [hunk 3]
```diff
@@ -63,17 +62,14 @@
...
-import circle2 from "@/assets/images/why/why-canopas-mobile-2.svg";
-import circle3 from "@/assets/images/why/why-canopas-mobile-3.svg";
-import circle4 from "@/assets/images/why/why-canopas-mobile-4.svg";
-import circle5 from "@/assets/images/why/why-canopas-mobile-5.svg";
-
+import circle1 from "@/assets/images/why/why-canopas-mobile-1.webp";
+import circle2 from "@/assets/images/why/why-canopas-mobile-2.webp";
+import circle3 from "@/assets/images/why/why-canopas-mobile-3.webp";
+import circle4 from "@/assets/images/why/why-canopas-mobile-4.webp";
+import circle5 from "@/assets/images/why/why-canopas-mobile-5.webp";
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/ServiceSectionMobile.vue:72]: Do space changes

File: vue-frontend/src/components/home-new/CTASection.vue [hunk 4]
```diff
@@ -24,12 +24,12 @@
         </h2>
         <div class="tw-flex tw-flex-wrap tw-justify-center">
           <router-link
-            class="tw-flex tw-items-center tw-rounded-[2rem] gradient-btn"
+            class="tw-flex tw-w-max tw-items-center tw-rounded-full tw-py-3 tw-px-3 gradient-btn consultation-btn"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/home-new/CTASection.vue:27]: Refactor `Talk to our experts` button also

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Comment by cp-sumi-k** [comment]
`pointer` changes not done in the blogs section

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-priyanka-g — DISMISSED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**